_includes/project-new.md

```diff
@@ -83,8 +84,9 @@ specific to this project and used by MISO to track the project internally.
 
 In order to make sure MISO is SRA-compliant, you must add a _Study_ to your Project. Studies are legacy objects that are not really used in the current system, but need to be added anyway.
 
-1. Click on the _Studies_ section to expand the section.
-1. Click _Add_ at the top left of the table.
+1. Click the _My Projects_ tab at the top of the MISO dashboard.
+1. Click on your Project _Name_ to expand it.
+1. Click _Add_ at the top left of the _Studies_ table.
```

Comment:
The expansion happens on the Studies section of the project page, not on the project page itself. What about:
```
1. Click the _My Projects_ tab at the top of the MISO dashboard.
1. Click on your Project _Name_.
1. If the _Studies_ table is not initially visible, click on the _Studies_ section to expand it.
1. Click _Add_ at the top left of the _Studies_ table.
```